.github/workflows/ci.yml

```diff
@@ -39,8 +39,8 @@ jobs:
       - name: Install Rust toolchain
         run: rustup component add rustfmt clippy
 
-      - name: Run clippy
-        run: cargo clippy --all-targets --all-features -- --deny warnings
+      - name: Run clippy without default features
+        run: cargo clippy --all-targets --no-default-features --features unstable,shared-memory -- --deny warnings
```

Comment:
if "without default features" then why `unstable` and `shared-memory` features are enabled?

Reply:
Default features are those of zenoh, so we do not really have any code behind them - so clippy will only lint zenoh code. On the other hand we have a lot of zenoh-c code behind unstable and shared-memory features which we want to lint.